tool_collections/kraken/kraken_report/kraken-mpa-report.xml

```diff
@@ -0,0 +1,68 @@
+<tool id="kraken-mpa-report" name="Kraken-mpa-report" version="1.1.0">
+    <description>
+        view report of classification for multiple samples
+    </description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <command>
+      <![CDATA[
+
+#for $input_count, $input_classification in enumerate( $classifications ):
+ ln -s "${input_classification.classification}" "sample_${input_count}" &&
+#end for
+
+kraken-mpa-report @INPUT_DATABASE@
+
+#for $input_count, $input_classification in enumerate( $classifications ):
+ "sample_${input_count}"
+#end for
+
+${show_zeros}
+${header_line}
+
+> "$output_report"
+
+
+	]]>
+    </command>
+    <inputs>
+      <repeat name="classifications" title="Classifications" min="1">
```

Comment:
@nekrut why is this so? It shouldn’t ... iteration over the `multiple=True` should be equivalent with repeat, afaik.